bom/pom.xml

```diff
@@ -41,11 +41,13 @@
         <protostuff.version>1.5.9</protostuff.version>
         <grpc.version>1.53.0</grpc.version>
 
+        <fury.version>0.2.0</fury.version>
```

Comment:
Fury 0.3.0 has been released https://github.com/alipay/fury/releases/tag/v0.3.0 , could you  update to latest version?

Reply:
sure